llvm/lib/Object/ELFObjectFile.cpp

```diff
@@ -833,10 +840,11 @@ std::vector<ELFPltEntry> ELFObjectFileBase::getPltEntries() const {
         consumeError(PltContents.takeError());
         return {};
       }
+
```

Comment:
Was this extra newline intentional? If not I recommend removing to keep the diff as small as possible